### PR TITLE
Tweaks Unconsciousness

### PR DESCRIPTION
**Pull request by SleepyGemmy**
this PR tweaks unconsciousness and lets people see (but still not hear) around their character in a small radius while unconscious, so that they don't have to stare at a black screen for 10 to 20 minutes. it also tweaks some sources of unconsciousness to make them more believable and immersive, as well as other clean code related changes.

**changes**
-
- lets people that are unconscious see around them in a 1.5 tile radius (including the tile they're on).
- removes paralysis from N2O as it already makes you unconscious.
- adds and tweaks eye blur on many different sources of unconsciousness to make it more clear you are unconscious.

**tldr**
-
<details>
<summary>click for image</summary>

image
</details>

### CODE REVIEW

**Comment by SleepyGemmy** [comment]
!review

**Review by Alberyk — APPROVED** [review]

File: code/modules/organs/internal/lungs.dm [hunk 1]
```diff
@@ -243,15 +242,10 @@
 	if(breath.gas[GAS_N2O])
 		var/SA_pp = (breath.gas[GAS_N2O] / breath.total_moles) * breath_pressure
 
-		// Enough to make us paralysed for a bit
-		if(SA_pp > SA_para_min)
-
-			// 3 gives them one second to wake up and run away a bit!
-			owner.Paralyse(3)
```
Comment (Arrow768) [inline code/modules/organs/internal/lungs.dm:250]: The removal should be in the changelog

**Review by Arrow768 — CHANGES_REQUESTED** [review]

File: code/modules/organs/internal/lungs.dm (hunk 1 above)
Comment (SleepyGemmy) [reply code/modules/organs/internal/lungs.dm:250]: addressed.

**Review by Arrow768 — APPROVED** [review]
